ffmpeg/ffmpeg.go

```diff
@@ -694,6 +694,24 @@ func createCOutputParams(input *TranscodeOptionsIn, ps []TranscodeOptions) ([]C.
 		if len(p.VideoEncoder.Name) <= 0 && len(p.VideoEncoder.Opts) <= 0 {
 			p.VideoEncoder.Opts = map[string]string{
 				"forced-idr": "1",
+				"preset":     "slow",
+				"tier":       "high",
+			}
+			if p.Profile.CRF != 0 {
+				if p.Profile.CRF <= 63 {
+					p.VideoEncoder.Opts["crf"] = strconv.Itoa(int(p.Profile.CRF))
+				} else {
+					glog.Warning("Cannot use CRF param, value out of range (0-63)")
+				}
+
+				// There's no direct numerical correspondence between CQ and CRF.
+				// From some experiments, it seems that setting CQ = CRF + 7 gives similar visual effects.
+				cq := p.Profile.CRF + 7
```

Comment:
How did you end up with this formula? Lower CQ values typically indicate better quality at expense of compressions. Also encoders typically will pick either CQ and CRF as the main control so I'm not sure if CQ here will actually make a difference.

Reply:
CRF is used by CPU-based transcoding, CQ is used by GPU-based transcoding.
So, only one of these values will be used at the time.

Technically there is no direct numerical correspondence between CQ and CRF, so the other option would be to set both CRF and CQ in our API. But I think it's not a pragmatic approach right now. Most of our transcoders are GPU anyway.

How did I come with `cq  := p.Profile.CRF + 7`, just experimenting with videos. But obviously not for all videos it will give the same results, since technically CRF and CQ values do not correspond to each other.

Reply:
The `cq  := p.Profile.CRF + 7` doesn't quite make sense to me esp since they don't correspond to one another. Also if CRF is CPU only, then why do we need it?

Reply:
Explained here: https://discord.com/channels/423160867534929930/1140578841136484402/1149041283583721634

lets discuss more in discord